crates/polars-plan/src/plans/schema.rs

```diff
@@ -32,6 +32,11 @@ impl DslPlan {
 #[derive(Clone, Debug, Default)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct FileInfo {
+    /// Schema of the physical file.
+    ///
+    /// Notes:
+    /// - Does not include logical columns like `include_file_path` and row index.
+    /// - Always includes all hive columns.
```

Comment:
Even if the file doesn't have these columns yet?

Reply:
Yup, it is rather weird behavior.